packages/example-log-extension/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "loopback4-example-log-extension",
+  "version": "1.0.0",
```

Comment:
example getting started is version as `4.0.0-alpha.0`. Should we use the same here?

Reply:
Sure.